index.js

```diff
@@ -59,10 +60,10 @@ const buildServices = async (inputDirPath, resultFilePath) => {
         const serviceFileNames = await getServicesFileNames(inputDirPath);
         await restoreRemovedInputServices(resultFilePath, serviceFileNames);
         await overwriteResultFile(inputDirPath, resultFilePath, serviceFileNames);
-        console.log('Successfully finished building services.json');
+        console.log(chalk.bold.green('Successfully finished building services.json'));
```

Comment:
let's create a file `scripts/logger.js` and add a helper for logging which can be called
`logger.success('good');`
`logger.error('bad');`

```
const logger = {
    info: (message) => { ... }, // just console.log()

    success: (message) => { ... }, // console.error + chalk

    error: (message) => { ... }, // console.log + chalk
};
```